projects/05-guess-the-number/docs/README-csharp.md

```diff
@@ -0,0 +1,423 @@
+# C# 💻
+
+## 1\. Preámbulo
```

Comment:
No debe ser necesario poner este `\.` con los headings

No hay en los otros markdowns [por ejemplo lo de python](https://github.com/Laboratoria/curriculum/blob/main/projects/05-guess-the-number/docs/README-python.md?plain=1)